acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/account/AccountManagementAcceptanceTest.java

```diff
@@ -16,22 +16,23 @@
 
 import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
 
+import java.io.IOException;
 import java.util.List;
 
 import org.junit.Test;
 
 public class AccountManagementAcceptanceTest extends AcceptanceTestBase {
 
   @Test
-  public void ethSignerAccountListHasSingleEntry() {
+  public void ethSignerAccountListHasSingleEntry() throws IOException {
     final List<String> accounts = ethSigner().accounts().list();
     assertThat(accounts.size()).isEqualTo(1);
     assertThat(ethNode().accounts().balance(accounts.get(0))).isNotNull();
     assertThat(accounts.get(0)).isEqualTo("0xfe3b557e8fb62b89f4916b721be55ceb828dbd73");
   }
 
   @Test
-  public void ethNodeAccountListIsEmpty() {
+  public void ethNodeAccountListIsEmpty() throws IOException {
```

Comment:
Is this Exception actually needed?

Reply:
accounts().list() needs it!

Reply:
After fixing the Accounts and Transactions classes this is no longer needed!